### PR TITLE
SIDE-1540 Add "markdown" attribute to disambiguation and list whispers

### PR DESCRIPTION
**Pull request by BrettMugglinOlive**

### CODE REVIEW

File: ldk/go/whisperContentDisambiguation.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import "github.com/open-olive/loop-development-kit/ldk/go/proto"
 
 type WhisperContentDisambiguation struct {
 	Label    string                                         `json:"label"`
+	Markdown string											`json:"markdown"`
```
Comment (bbrewer-oliveai) [inline ldk/go/whisperContentDisambiguation.go:7]: Pretty sure golint is gonna freak out about this.

**Review by bbrewer-oliveai — APPROVED** [review]
Looks good though I'm curious how/if you tested this. Manually with a self-test loop is an acceptable answer.

File: ldk/go/whisperContentDisambiguation.go (hunk 1 above)
Comment (BrettMugglinOlive) [reply ldk/go/whisperContentDisambiguation.go]: Yeah, you're right. I go fmted the code

File: ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/IWhisperService.cs [hunk 2]
```diff
@@ -143,6 +143,11 @@ public struct WhisperList
         /// </summary>
         public WhisperConfig Config;
 
+        /// <summary>
+        /// Markdown formatted string displayed at the top of the form.
+        /// </summary>
+        public string Markdown;
```
Comment (richseviora) [inline ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/IWhisperService.cs:149]: I think this also requires a corresponding change to `ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/WhisperFormBuilder.cs:26`

**Review by richseviora — CHANGES_REQUESTED** [review]
Needs an additional change in .net

File: ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/IWhisperService.cs (hunk 2 above)
Comment (BrettMugglinOlive) [reply ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/IWhisperService.cs:149]: If this is a change for just WhisperList, would I need to update the form builder?

Comment (richseviora) [reply ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/IWhisperService.cs:149]: Yes; the `WhisperFormBuilder` also has a `BuildRequest(WhisperList list, Proto.Session session)` method where the message is constructed.

**Review by richseviora — APPROVED** [review]

**Review by bbrewer-oliveai — APPROVED** [review]

**Pull request merged**